source/sass/components/nav.scss

```diff
@@ -234,6 +234,22 @@
 }
 
 #multipage-nav {
+  .multipage-login-restricted::after {
+    content: '🙈';
+  }
+
+  .multipage-groups-restricted::after {
+    content: '👥';
+  }
+
+  .multipage-password-restricted::after {
+    content: '🔒';
+  }
+
+  .multipage-draft::after {
+    content: '🐣';
```

Comment:
not quite seeing the connection to "draft", but frankly I also don't care enough to go "no! fix this!" =D

Reply:
It’s “partially hatched” 😂

It was Matthew's suggestion 🙂